### PR TITLE
[INLONG-6216][Manager] Support getting group topic info

### LINKED ISSUE #6216: [Feature][Manager] Support get group topic info
### Description

_No response_

### Use case

_No response_

### Are you willing to submit PR?

- [X] Yes, I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by vernedeng**
### Prepare a Pull Request

- Fixes #6216

### Motivation

*Explain here the context, and why you're making that change. What is the problem you're trying to solve?*

### Modifications

*Describe the modifications you've done.*

### CODE REVIEW

File: inlong-manager/manager-dao/src/main/java/org/apache/inlong/manager/dao/mapper/InlongClusterEntityMapper.java [hunk 1]
```diff
@@ -57,4 +57,6 @@ List<InlongClusterEntity> selectByKey(@Param("clusterTag") String clusterTag, @P
 
     int deleteByPrimaryKey(Integer id);
 
+    List<InlongClusterEntity> selectByClusterTag(String clusterTag);
```
Comment (healchow) [inline inlong-manager/manager-dao/src/main/java/org/apache/inlong/manager/dao/mapper/InlongClusterEntityMapper.java:60]: Please move the `select` method after the `selectAllClusters()` method.

File: inlong-manager/manager-pojo/src/main/java/org/apache/inlong/manager/pojo/cluster/ClusterBriefInfo.java [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+/**
+ * Inlong cluster brief info
+ */
+@Data
+@SuperBuilder
+@NoArgsConstructor
+@AllArgsConstructor
+@ApiModel("Inlong cluster brief info")
+@JsonTypeInfo(use = JsonTypeInfo.Id.NAME, visible = true, property = "type")
+public abstract class ClusterBriefInfo {
```
Comment (healchow) [inline inlong-manager/manager-pojo/src/main/java/org/apache/inlong/manager/pojo/cluster/ClusterBriefInfo.java:37]: Suggest reusing the `ClusterInfo` class, and its sub-classes.

File: inlong-manager/manager-dao/src/main/resources/mappers/InlongClusterEntityMapper.xml [hunk 3]
```diff
@@ -247,4 +247,14 @@
         where id = #{id,jdbcType=INTEGER}
     </delete>
 
+    <select id="selectByClusterTag" resultType="org.apache.inlong.manager.dao.entity.InlongClusterEntity">
+        select
+        <include refid="Base_Column_List"/>
+        from inlong_cluster
+        <where>
+            is_deleted = 0
+            and inlong_cluster_tag = #{clusterTag, jdbcType=STRING}
```
Comment (healchow) [inline inlong-manager/manager-dao/src/main/resources/mappers/InlongClusterEntityMapper.xml:256]: Did it work for the type `STRING`?

File: inlong-manager/manager-dao/src/main/java/org/apache/inlong/manager/dao/mapper/InlongStreamExtEntityMapper.java [hunk 4]
```diff
@@ -44,6 +44,9 @@ public interface InlongStreamExtEntityMapper {
 
     List<InlongStreamExtEntity> selectByRelatedId(@Param("groupId") String groupId, @Param("streamId") String streamId);
 
+    InlongStreamExtEntity selectByKey(@Param("groupId") String groupId, @Param("streamId") String streamId,
+                                      @Param("keyName") String keyName);
```
Comment (healchow) [inline inlong-manager/manager-dao/src/main/java/org/apache/inlong/manager/dao/mapper/InlongStreamExtEntityMapper.java:48]: Not necessary to indent.

File: inlong-manager/manager-pojo/src/main/java/org/apache/inlong/manager/pojo/cluster/ClusterBriefInfo.java (hunk 2 above)
Comment (vernedeng) [reply inlong-manager/manager-pojo/src/main/java/org/apache/inlong/manager/pojo/cluster/ClusterBriefInfo.java]: fixed, thx

File: inlong-manager/manager-dao/src/main/java/org/apache/inlong/manager/dao/mapper/InlongClusterEntityMapper.java (hunk 1 above)
Comment (vernedeng) [reply inlong-manager/manager-dao/src/main/java/org/apache/inlong/manager/dao/mapper/InlongClusterEntityMapper.java:60]: fixed, thx

File: inlong-manager/manager-dao/src/main/resources/mappers/InlongClusterEntityMapper.xml (hunk 3 above)
Comment (vernedeng) [reply inlong-manager/manager-dao/src/main/resources/mappers/InlongClusterEntityMapper.xml:256]: my fault, fixed, thx

File: inlong-manager/manager-dao/src/main/java/org/apache/inlong/manager/dao/mapper/InlongStreamExtEntityMapper.java (hunk 4 above)
Comment (vernedeng) [reply inlong-manager/manager-dao/src/main/java/org/apache/inlong/manager/dao/mapper/InlongStreamExtEntityMapper.java:48]: fixed, thx

**Review by vernedeng — COMMENTED** [review]
lgtm

**Review by woofyzhao — APPROVED** [review]

**Review by dockerzhang — APPROVED** [review]

**Review by luchunliang — APPROVED** [review]
LGTM

**Review by gosonzhang — APPROVED** [review]

**Review by healchow — APPROVED** [review]

**Pull request merged**